.gitlab-ci.yml

```diff
@@ -61,6 +61,8 @@ RPM:
       - RUNNER:
           - aws/rhel-8.5-nightly-x86_64
```

Comment:
rhel-8.5 should go away in as a follow-up, we don't plan any further development for it

Reply:
I figured we can keep 8.5 nightly until GA in case something unusual crops up.  Might be wasteful CI though.

Reply:
Might restrict it to the 8.5.0 branch? (I also don't mind just keeping it)

Reply:
You have a good point Ondrej. Most probably none of these things will happen until GA 😄